Tools/scripts/summarize_stats.py

```diff
@@ -483,7 +484,7 @@ def get_optimization_stats(self) -> dict[str, tuple[int, int | None]]:
             ): (trace_too_long, attempts),
             Doc(
                 "Trace too short",
-                "A potential trace is abandoced because it it too short.",
+                "A potential trace is abandoned because it it too short.",
```

Comment:
I hate to be nitpicky, but please remove unrelated typo fixes like this from the PR. It makes it harder to review :(

Reply:
No problem.
should I open another PR just for that? What would be the best way to fix that?

Reply:
Yeah, go ahead and open a new PR. Tag me and I'll apply `skip news` and `skip issue`.